web/frontend/angular/src/app/auth/auth.service.ts

```diff
@@ -15,6 +17,11 @@ export class AuthService {
         localStorage.removeItem("ACCESS_TOKEN")
     }
     public register(userData: User) {
-        //TODO
+        userData.username = userData.email
```

Comment:
The username and the email are distinctly separate fields. It is not appropriate to make the email the username.